tests/test_class_rules.py

```diff
@@ -0,0 +1,78 @@
+import dash_ag_grid as dag
+from dash import Dash, html, dcc
+from . import utils
+import time
+from dash.testing.wait import until
+
+def test_cr001_class_rules(dash_duo):
+    app = Dash(__name__,
+                    meta_tags=[{'http-equiv': 'content-security-policy',
+                                'content': "default-src 'self'; script-src 'self' 'unsafe-inline';"
+                                           " style-src https://* 'self' 'unsafe-inline'; "
+                                           "font-src data: https://* 'self' 'unsafe-inline';"
+                                           "img-src data: https://* 'self'"}],
+                    )
+
+    rowData = [
+        {"make": "Toyota", "model": "Celica", "price": 35000},
+        {"make": "Ford", "model": "Mondeo", "price": 32000},
+        {"make": "Porsche", "model": "Boxter", "price": 72000},
+    ]
+
+    columnDefs = [
+        {"headerName": "Make", "field": "make", "cellClassRules": {"orange": "params.value == 'Ford'",
+                                                                                     "red": "params.value == 'Porsche'",
+                                                                                     "blue": "params.value == 'Toyota'"}},
+        {"headerName": "Model", "field": "model"},
+        {"headerName": "Price", "field": "price"},
+    ]
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown(
+                "Try filtering the data in the grid using the inline filters (click the hamburger menu in each column). Though the data is still in `rowData`, you can view the virtual row data in callbacks by watching the `virtualRowData` property."
+            ),
+            dag.AgGrid(
+                id="grid",
+                columnDefs=columnDefs,
+                rowData=rowData,
+                defaultColDef={
+                    "editable": True,
+                },
+                rowClassRules={"orange": "params.data.make == 'Ford'",
+                               "red": "params.data.make == 'Porsche'",
+                               "blue": "params.data.make == 'Toyota'"},
```

Comment:
For clarity, can we give the classes in `rowClassRules` different names & functions from the ones in `columnDefs`?

Reply:
Sure, I'm adding `row-` in front of them.

Reply:
Also made the functions query the model instead of the make

Reply:
@alexcjohnson :)